ubuntu-browser-shell-container/release.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -eu
+
+DOCKER_PS_RESULT=`docker ps > /dev/null 2>&1`;
```

Shellcheck (0.10.0)

[warning] DOCKER_PS_RESULT appears unused. Verify use (or export if used externally).

(SC2034)